imagersite/imager_profile/models.py

```diff
@@ -0,0 +1,49 @@
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+from django.db.models.signals import post_delete
+from django.dispatch import receiver
+
+
+class ActiveProfileManager(models.Manager):
+    def get_query_set(self):
+        return super(ActiveProfileManager, self).get_query_set().filter(
+            user__is_active=True)
+
+
+@python_2_unicode_compatible
+class ImagerProfile(models.Model):
+    user = models.OneToOneField(User, related_name='profile', null=False)
+    camera = models.CharField(max_length=128,
+        help_text='What is the make and model of your camera?')
+    address = models.TextField()
+    website = models.URLField()
```

Comment:
these fields will be required.  In order to allow them to be optional, you'd have to set `blank`=True for each.